### PR TITLE
Sils/readm

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -107,6 +107,9 @@ to provide helpful libraries and provides multiple user interfaces.
 One could say coala unites all language independent parts of a linter
 and is arbitrarily extensible.
 
+To see what coala can do for you and your language, take a look at
+`our capabilities listing <https://github.com/coala-analyzer/bear-docs>`__.
```
Comment (hypothesist) [inline docs/index.rst]: Would rather link to the README.rst directly IMO - it gives a full view list like an actual website.

**Comment by Adrianzatreanu** [comment]
ack ad004ace

File: docs/index.rst (hunk 1 above)
Comment (Adrianzatreanu) [inline docs/index.rst]: agreed, link should be https://github.com/coala-analyzer/bear-docs/blob/master/README.rst

**Comment by Adrianzatreanu** [comment]
unack 30c197f9f

**Comment by Adrianzatreanu** [comment]
unack ad004ac it should have the link to readme as well imo

**Comment by Adrianzatreanu** [comment]
ack 6de3296 5c2f289

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala/pull/2340/commits/5c2f28910c17cf1f3e201e2dc781840773261a77

**Comment by Adrianzatreanu** [comment]
rebaaaaase @sils1297 :D

**Comment by hypothesist** [comment]
... and reack both and merge :)

**Comment by sils1297** [comment]
ack
cedf8cc docs: Add supported languages link
167d780 README: Add link to bear docs

**Comment by sils1297** [comment]
(

**Comment by sils1297** [comment]
re)

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**